packages/sui-studio/src/components/demo/index.js

```diff
@@ -150,7 +159,7 @@ export default class Demo extends Component {
 
         <Preview
           code={playground}
-          scope={{React, [`${cleanDisplayName(Enhance.displayName)}`]: Enhance, domain}}
+          scope={{React, [`${cleanDisplayName(Enhance.displayName)}`]: Enhance, domain, ...nonDefaultExports}}
```

Comment:
Are you try what happen when `nonDefaultExports ` is undefined ?!

Reply:
yes, it still works